ManageWikiSettings.php

```diff
@@ -297,6 +297,47 @@
 		'help' => 'This sets the web client to use. If you are not using Freenode, select Other Server.',
 	],
 
+	// CommentStream
+	'wgCommentStreamsNewestStreamsOnTop' => [
+		'name' => 'Order of comments',
+		'from' => 'CommentStream',
+		'restricted' => false,
+		'type' => 'check',
+		'overridedefault' => true,
+		'section' => 'CommentStream',
+		'help' => 'Comments streams will be displayed with the newest comment stream on top',
```

Comment:
Remove the weird streams words in this sentience.

Reply:
Is that okay or is there something else missing? Thank you so much for your help.

Reply:
It's ok.